src/components/OverlayImage.tsx

```diff
@@ -39,6 +39,28 @@ const OverlayImage = ({ url, opacity, scaling, isLocked, showDifference }: Overl
     return () => window.removeEventListener("mousemove", handleMouseMove);
   }, [mouseDown]);
 
+  // Ability to move zeplin overlay with Shift + Arrow keys
+  useEffect(() => {
+    const handleKeyDown = (event) => {
```

Comment:
We need to respect `isLocked` here. If it is true, we shouldn't update position.

Reply:
Added check for ```isLocked``` property.